README.md

```diff
@@ -1,165 +1,268 @@
+
+# How to start using Mocky
+
+Mocks generation is based on `mocky.yml` file.
+
+First, create file in your project root directory with following structure:
+
+```yml
+sources:
+  - ./ExampleApp
+  - ./ExampleAppTests
+templates:
+  - ./Pods/Mocky/Mocky/Templates
+output:
+  ./ExampleApp
+args:
+  testable:
+    - ExampleApp
+  import:
+    - RxSwift
+    - RxBlocking
 ```
-// sourcery: mock = "ItemsModel"
-class ItemsModelMock: ItemsModel, Mock {
-// sourcery:inline:auto:ItemsModelMock.autoMocked
-    //swiftlint:disable force_cast
-
-    var invocations = [MethodType]()
-    var methodReturnValues: [MethodProxy] = []
-
-    //MARK : ItemsModel  
-
-    func getExampleItems() -> Observable<[Item]> {
-        addInvocation(.getExampleItems)
-        return methodReturnValue(.getExampleItems) as! Observable<[Item]> 
-    }
-    
-    func getItemDetails(item: Item) -> Observable<ItemDetails> {
-        addInvocation(.getItemDetails(item: .value(item)))
-        return methodReturnValue(.getItemDetails(item: .value(item))) as! Observable<ItemDetails> 
-    }
-    
-    enum MethodType: Equatable {
-
-        case getExampleItems    
-        case getItemDetails(item : Parameter<Item>)     
-    
-        static func ==(lhs: MethodType, rhs: MethodType) -> Bool {
-            switch (lhs, rhs) {
-
-                case (.getExampleItems, .getExampleItems): return true                
-                case (let .getItemDetails(lhsParams), let .getItemDetails(rhsParams)): return lhsParams == rhsParams                 
-                default: return false   
-            }
-        }
-    }
-
-    struct MethodProxy {
-        var method: MethodType 
-        var returns: Any? 
-
-        static func getExampleItems(willReturn: Observable<[Item]>) -> MethodProxy {
-            return MethodProxy(method: .getExampleItems, returns: willReturn)
-        }
-        
-        static func getItemDetails(item: Parameter<Item>, willReturn: Observable<ItemDetails>) -> MethodProxy {
-            return MethodProxy(method: .getItemDetails(item: item), returns: willReturn)
-        }
-         
-    }
-
-    private func methodReturnValue(_ method: MethodType) -> Any? {
-        let all = methodReturnValues.filter({ proxy -> Bool in
-            return proxy.method == method
-        })
-
-        return all.last?.returns
-    }
-// sourcery:end
+
++ **sources**: all directories you want to scan for protocols/files marked to be auto mocked, or inline mocked
++ **templates**: path to Mocky sourcery templates, in Pods directory
++ **output**: place where `Mock.generated.swift` will be placed
++ **testable**: specify imports for Mock.generated, that should be marked as `@testable` (usually tested app module)
++ **import**: all additional imports, external libraries etc. to be placed in Mock.generated
+
+## Generate mocks:
+
+1. **manually**: by triggering:
+
+  `Pods/Sourcery/bin/sourcery --config <your config yml path> --disableCache`
```

Comment:
we shouldn't disable cache for mocks generation, it was only for template development.